simple/sample/output/observations_countries.csv

```diff
@@ -1,14 +1,14 @@
-dcid,date,var1,var2
-country/AFG,2023,0.19,6
-country/YEM,2023,0.21,56
-country/AGO,2023,0.29,6
-country/ZMB,2023,0.31,34
-country/ZWE,2023,0.37,76
-country/ALB,2023,0.5,34
-wikidataId/Q22062741,2023,0.5,97
-country/DZA,2023,0.52,92
-country/AND,2023,0.76,9
-country/ASM,2023,,34
-country/AIA,2023,,42
-country/WLF,2023,,75
-country/ESH,2023,,65
+dcid,date,var1,var2,provenance
+country/AFG,2023,0.19,6,custom/p/Prov_1
```

Comment:
from data size optimization perspective, maybe have shorter provenance id, since there is already /p to denote provenance so "custom/p/1" would be good? Same applied to custom/s/1. Also wonder should make "custom" shorter as "_". 

So we will have "_/p/1" and "_/s/1"

@pradh thoughts?

Reply:
how about `c/s/` ?

Reply:
c/ sg and we can make sure not using it in base DC.

Reply:
Done. Using `c/s/` prefix for sources and `c/p/` for provenances now.